src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1012,6 +1018,21 @@ void goto_instrument_parse_optionst::instrument_goto_program()
   else
     options.set_option("assert-to-assume", false);
 
+  if(cmdline.isset("use-abstraction"))
+  {
+    std::string abs_file = cmdline.get_value("use-abstraction");
```

Comment:
If this code is common to both CBMC and goto-instrument, why not put it all in a function so that there is not a parallel maintainence issue?

Reply:
Removed the option from CBMC completely.